tsconfig.base.json

```diff
@@ -15,7 +15,7 @@
     "forceConsistentCasingInFileNames": true,
     "useDefineForClassFields": false,
     "jsx": "react-jsx",
-    "lib": ["ES6", "dom"], // "ES6", "DOM" | "es2015", "dom"
+    "lib": ["ES2020", "dom"],
```

Comment:
Does ES2021 work here?  If so, please use ES2021 instead.

Reply:
It is more complicated than that, we should test each

Reply:
https://kangax.github.io/compat-table/es2016plus/

Reply:
> Does ES2021 work here? If so, please use ES2021 instead.

Check my other PR (which is still not quite ready) for dropping lodash.clonedeep - #517 . I changed it to ES2021 there because older versions don't support structuredClone. Once that gets merged we'll target ES2021

Reply:
Based on my research, the only thing that might be tricky is `Function.prototype.toString revision` that doesn't seem to be fully supported on Safari or iOS. What are your take @StefanNedelchev @wayfarer3130 

![image](https://user-images.githubusercontent.com/7490180/228615853-90f9b212-18b5-490f-b881-4d93f7a345c8.png)

Reply:
> Based on my research, the only thing that might be tricky is `Function.prototype.toString revision` that doesn't seem to be fully supported on Safari or iOS. What are your take @StefanNedelchev @wayfarer3130
> 
> ![image](https://user-images.githubusercontent.com/7490180/228615853-90f9b212-18b5-490f-b881-4d93f7a345c8.png)

do we rely on this support?